@kof I just committed the [styled-jss branch](https://github.com/DSchau/css-in-js-playground/compare/styled-jss?expand=1). If you'd like you can check it out yourself and see the issues.

My first instinct is that I need to configure a Webpack loader differently, _or_ it could be caused by using React 16. Here is the (first) error:

```
Uncaught (in promise) Error: Cannot find module "react-dom/lib/DOMProperty"
    at webpackMissingModule (index.js:7)
    at Object.398 (index.js:7)
    at __webpack_require__ (bootstrap fb3149dccdb8137fc7cf:705)
    at fn (bootstrap fb3149dccdb8137fc7cf:111)
    at Object.397 (filterProps.js:7)
    at __webpack_require__ (bootstrap fb3149dccdb8137fc7cf:705)
    at fn (bootstrap fb3149dccdb8137fc7cf:111)
    at Object.396 (styled.js:11)
    at __webpack_require__ (bootstrap fb3149dccdb8137fc7cf:705)
    at fn (bootstrap fb3149dccdb8137fc7cf:111)
```

and the second

```
client:139 ./node_modules/is-react-prop/lib/index.js
Module not found: Error: Can't resolve 'react-dom/lib/EventPluginRegistry' in '/Users/dschau/Code/Personal/css-in-js/playground/node_modules/is-react-prop/lib'
 @ ./node_modules/is-react-prop/lib/index.js 11:27-71
 @ ./node_modules/styled-jss/utils/filterProps.js
 @ ./node_modules/styled-jss/styled.js
 @ ./node_modules/styled-jss/createStyled.js
 @ ./node_modules/styled-jss/index.js
 @ ./src/utils/libraries.ts
 @ ./src/components/CodePreview/index.tsx
 @ ./src/components/CodeProvider/index.tsx
 @ ./src/App.tsx
 @ ./src/index.tsx
 @ multi (webpack)-dev-server/client?http://0.0.0.0:8000 webpack/hot/dev-server ./src/index
```

--- a/package.json
+++ b/package.json
@@ -59,7 +59,6 @@
     "glamorous": "^4.7.0",
     "history": "^4.6.3",
     "jss": "^8.1.0",
-    "jss-extend": "^5.0.0",
     "jss-preset-default": "^3.0.0",
     "lodash.debounce": "^4.0.8",
     "lodash.kebabcase": "^4.1.1",

--- a/src/constants/snippets.ts
+++ b/src/constants/snippets.ts
@@ -5,11 +5,4 @@ import * as Cxs from '../snippets/cxs';
 import * as Radium from '../snippets/radium';
 import * as Jss from '../snippets/jss';
 
-export {
-  StyledComponents,
-  Glamorous,
-  Aphrodite,
-  Cxs,
-  Radium,
-  Jss
-};
+export { StyledComponents, Glamorous, Aphrodite, Cxs, Radium, Jss };

--- a/src/snippets/jss.js
+++ b/src/snippets/jss.js
@@ -5,7 +5,6 @@ import extend from 'jss-extend';
 import Logo from './logo';
 
 jss.setup(preset());
-jss.setup(extend());
 
 const styles = {
   container: {

--- a/src/utils/libraries.ts
+++ b/src/utils/libraries.ts
@@ -42,14 +42,11 @@ export default code => {
   } else if (matches('jss')) {
     return Promise.all([
       import('jss'),
-      import('jss-preset-default'),
-      import('jss-extend')
-    ])
-      .then(([jss, preset, extend]) => ({
-        jss: (jss as any).default,
-        preset: preset.default,
-        extend: (extend as any).default
-      }));
+      import('jss-preset-default')
+    ]).then(([jss, preset]) => ({
+      jss: (jss as any).default,
+      preset: preset.default
+    }));
   }
   return Promise.resolve({});
 };

